lib/http.js

```diff
@@ -107,7 +107,7 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
       if(meterId !== record.meter.id) {
         throw new BedrockError(
           'The meter id in request body does not match its record.',
-          'DataError', {
+          'URLMismatchError', {
```

Comment:
Please also add `requestUrl` and `meterId` here. And rename `expectedMeterId` to `expected` and `actualMeterId` to `actual`. This matches what we do in other services and it will preserve the properties from the other error that was removed.

Reply:
Updated here 3818f6d